templates/actor/effects.hbs

```diff
@@ -41,15 +43,15 @@
         <div class="effect-duration">{{effect.duration.label}}</div>
         <div class="effect-controls flexrow">
           {{#if @root.editable}}
-          <a class="effect-control" data-action="toggleEffect" title="{{localize " DRAW_STEEL.Effect.Toggle"}}">
+          <a class="effect-control" data-action="toggleEffect" data-tooltip="DRAW_STEEL.Effect.Toggle">
             <i class="fas {{#if effect.disabled}}fa-check{{else}}fa-times{{/if}}"></i>
           </a>
           {{/if}}
-          <a class="effect-control" data-action="viewDoc" title="{{localize " DOCUMENT.Update" type="Effect" }}">
+          <a class="effect-control" data-action="viewDoc" data-tooltip="{{localize "DOCUMENT.Update" type="Effect" }}">
```

Comment:
I've found that automatic linters sometimes choke on these nested quotes in handlebars, so we could consider changing to apostrophes on the inner strings.
```suggestion
          <a class="effect-control" data-action="viewDoc" data-tooltip="{{localize "DOCUMENT.Update" type="Effect"}}">
```